test/absinthe/type/object_test.exs

```diff
@@ -5,6 +5,7 @@ defmodule Absinthe.Type.ObjectTest do
 
   defmodule Schema do
     use Absinthe.Schema
+    use Absinthe.Fixture
```

Comment:
@maartenvanvliet Did this solve a warning or something?

Reply:
It causes a test failure when the tests are run with PERSISTENT_TERM=true